data/CollegeHero.sql

```diff
@@ -194,6 +194,35 @@ CREATE PROCEDURE getAllSectionInfoBySubject(IN subject VARCHAR(45))
     WHERE class.subject = subject;
   END//
 
+DROP PROCEDURE IF EXISTS getAllStudentOwingTuition;//
```

Comment:
I think it should only have // and no semicolon. Have you run this successfully yet?